modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -0,0 +1,46 @@
+= B2B Solutions and Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== B2B Solutions and Overview
+
+B2B (Business-to-Business) is the use of API's or Electronic Data Interchange (EDI) by enterprises to carry out business transactions such as purchasing goods from one another. B2B messages are typically sent over transports such as HTTP(s), FTP, FTPS, SFTP, JMS, and exchanges such as AS2. MuleSoft's B2B solutions integrate seamlessly into the Mule runtime.
+
+This overview introduces MuleSoft's B2B solution and links to pages that provide B2B implementation examples in the context of Anypoint Studio, Mule’s graphical user interface. In addition, provides links to configuration details for implementing Anypoint B2B in Anypoint Studio’s XML Editor.
+
+== Anypoint Partner Manager
+
+Through its web interface, Anypoint Partner Manager enables effective setup and management of a large number of trading partners. Additionally, via the ability to track and report on transaction progress, Partner Manager drives greater operational efficiency while managing B2B transactions. With Anypoint Partner Manager you can:
+
+* Perform technical on-boarding of partners.
+* Set up message flows to receive and send messages between your backend systems and your partners.
+* View and monitor how partner messages are being processed in the system in case of failure in order to take corrective action.
```

Comment:
Maybe this sentence can be modified to make it shorter, avoid the passive voice, and avoid using "in" so many times. Perhaps something like: "View and monitor how the system processes partner messages in case of failure, to take corrective action"

Reply:
Rewritten per Mysti.